.gitignore

```diff
@@ -72,4 +72,4 @@ GCMessengerSDKSample/.DS_Store
 .DS_Store
 */.DS_Store
 */GoogleService-Info.plist
-GoogleService-Info.plist
\ No newline at end of file
+GoogleService-Info.plist
```

Comment:
This should be in Master

Reply:
Yes. This is a master to develop PR. So it means it is in master, and we will add it to develop